project/spreadsheet.py

```diff
@@ -60,21 +52,208 @@ def writeHeaderColumnNamesToSheet(SPREADSHEET_ID, service, email):
         'valueInputOption': "RAW",
         'data': data
     }
-    valuesUpdated = service.spreadsheets().values().batchUpdate(
-        spreadsheetId=SPREADSHEET_ID, body=body).execute()
 
-    return valuesUpdated
+    cont = True
+
+    try:
+        service.spreadsheets().values().batchUpdate(spreadsheetId=SPREADSHEET_ID, body=body).execute()
 
-def writeRowsToSheet():
+    except googleapiclient.errors.HttpError:
+        print("Error. Could not create a spreadsheet for a user " + email + ". The value list might be out of index range ")
```

Comment:
It might be useful to print out the value list here - to help determine what went wrong. Same thing for every place you print this error.